docs/docs/references/configuration/cli/trivy_image.md

```diff
@@ -81,6 +81,7 @@ trivy image [flags] IMAGE_NAME
       --removed-pkgs                   detect vulnerabilities of removed packages (only for Alpine)
       --report string                  specify a format for the compliance report. (default "summary")
       --reset                          remove all caches and database
+      --runtime strings                Runtime(s) to use (docker,containerd,podman,remote) (default [docker,containerd,podman,remote])
```

Comment:
```suggestion
      --runtime strings                Runtime(s) to use (docker,containerd,podman,remote) (default priority [docker,containerd,podman,remote])
```

Reply:
We can set multiple runtimes
```suggestion
      --runtimes strings                Runtime(s) to use (docker,containerd,podman,remote) (default [docker,containerd,podman,remote])
```

Reply:
Both of these are taken care of.

The format `(default <...>)` in the CLI help is unalterable. To keep docs consistent with the CLI help, I have the following:
```
Runtime(s) to use, in priority order (docker,containerd,podman,remote) (default [docker,containerd,podman,remote])
```